tests/test_synced_collection.py

```diff
@@ -279,7 +349,7 @@ def test_call(self, synced_dict, testdata):
     def test_reopen(self, synced_dict, testdata):
         key = 'reopen'
         synced_dict[key] = testdata
-        synced_dict2 = deepcopy(synced_dict)
+        synced_dict2 = copy(synced_dict)
```

Comment:
`deepcopy` try to copy 'redis client' which raises error so I changed it to `copy`.

Reply:
That changes the test fundamentally, not a good idea. Instead try to implement the `RedisDict.__deepcopy__()` function. Please see [here](https://docs.python.org/3.7/library/copy.html) for more information on how to do this.